src/Renderer/Shader/Chunk/WebGL2_pars_fragment.glsl

```diff
@@ -0,0 +1,25 @@
+#if defined(WEBGL2)
```

Comment:
Could you link the `WebGLProgram` source on this ?

Reply:
comment added